recipe/meta.yaml

```diff
@@ -35,9 +52,12 @@ requirements:
     - git
     - ninja
   host:
-    - libcublas-dev  # [(cuda_compiler_version or "").startswith("12")]
+    - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
+    - libcublas-dev    # [(cuda_compiler_version or "").startswith("12")]
   run:
     - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+    - cuda-cudart  {{ cuda_compiler_version }}  # [(cuda_compiler_version or "").startswith("12")]
+    - cuda-cudart_{{ target_platform }} {{ cuda_compiler_version }}  # [(cuda_compiler_version or "").startswith("12") and win]
```

Comment:
Something Is wrong with the Windows dependencies. Unless I add the line above I get the following error on Windows

```
  ERROR (llama.cpp,bin/main.exe): Needed DSO Library/bin/cudart64_12.dll found in ['cuda-cudart_win-64']
  ERROR (llama.cpp,bin/main.exe): .. but ['cuda-cudart_win-64'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

Reply:
Yeah this is due to upstream issue ( https://github.com/conda-forge/cuda-cudart-feedstock/issues/13#event-11242601478 )

We just merged a fix ( https://github.com/conda-forge/cuda-cudart-feedstock/pull/14 ). So this should be resolved with the new packages

Think we can drop this then

Reply:
Raised issue ( https://github.com/conda-forge/llama.cpp-feedstock/issues/16 ) to track